docs/imu.md

```diff
@@ -0,0 +1,16 @@
+## IMU Sensor
+The IMU sensor in FSDS is using AirSim's built in IMU sensor simulation, that has been modelled and parametrized according to MPU 6000 IMU from [InvenSense](https://www.invensense.com/wp-content/uploads/2015/02/MPU-6000-Datasheet1.pdf)
+
+
+The maximum achievable internal IMU frequency is 1000 Hz, ouputing information of the vehicle's 3-axis angular velocity, 3-axis linear acceleration, as well as its orientation in quaternions. 
+
+IMU gyroscope and accelomter bias and accuracy parameters can be found and fine-tuned in AirLib/include/sensors/imu/ImuSimpleParams.hpp, or can be initialized with custom parameters in your settings.json file. 
```

Comment:
I guess we want to fix these so that they are the same for all teams right?

Reply:
I propose we use the default setting for the competition. In the integration handbook we can specify this.